ingest/rules/fetch_from_ncbi.smk

```diff
@@ -97,6 +97,9 @@ rule format_ncbi_dataset_report:
             --fields {params.ncbi_datasets_fields:q} \
             --elide-header \
             | csvtk add-header -t -l -n {params.ncbi_datasets_fields:q} \
+            | csvtk rename -t -f accession -n accession_version \
+            | csvtk -tl mutate -f accession_version -n accession -p "^(.+?)\." \
```

Comment:
I would refrain from adding more csvtk to any nextstrain code base when used with TSV files, and phase it out

There are rare but disastrous bugs when tsv fields contain single or double quotes: https://github.com/shenwei356/csvtk/issues/130

csvtk is just not a TSV tool, it mostly works, but it disobeys the standard.